src/clients/__tests__/index.ts

```diff
@@ -0,0 +1,20 @@
+export type MockMutator = {
+  setClientInitPayload: (resolve: {} | null, reject: {} | null) => void;
+  setTokenParsed: (props: {}) => void;
+  setUser: (props: {}) => void;
+  getInitCallCount: () => number;
+  getCreationCount: () => number;
+  resetMock: () => void;
+  setLoadProfilePayload: (resolve: {} | null, reject: {} | null) => void;
+  getLoginCallCount: () => number;
+  getLogoutCallCount: () => number;
+  setTokens: (newTokens: {}) => {};
+  getInstance: () => Keycloak.KeycloakInstance;
+  createValidUserData: (props?: {}) => { email: string; session_state: string };
+};
+
+describe('File must return one test ', () => {
+  it(' and this is just a note:', () => {
+    expect(!!'skip').toBeTruthy();
+  });
+});
```

Comment:
Is this kind of stuff **really** needed? Looks pretty 💩

Reply:
Jest test file filter cannot be changed via package.json, so made a new __mocks__-folder for these. Same logic is in profile-ui